test/statisticalmodel.jl

```diff
@@ -13,6 +13,7 @@ StatsAPI.deviance(::MyStatisticalModel) = 25
 StatsAPI.nulldeviance(::MyStatisticalModel) = 40
 StatsAPI.dof(::MyStatisticalModel) = 5
 StatsAPI.nobs(::MyStatisticalModel) = 100
+StatsAPI.momentmatrix(::MyStatisticalModel) = [1 2; 3 4; 5 6; 7 8]
```

Comment:
It's not really useful to define this. Existing methods in this file are defined just so that we can test the definitions for `aic`, etc. which are defined in this package. But it's not necessary to test empty function definitions.

Reply:
make sense.